.devcontainer/Dockerfile

```diff
@@ -17,5 +17,5 @@ RUN if [ "${REINSTALL_CMAKE_VERSION_FROM_SOURCE}" != "none" ]; then \
 #
 # Install apt packages which are required for builing libCZI (and CZICmd) with all features
  RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
-     && apt-get -y install --no-install-recommends libcurl4-openssl-dev rapidjson-dev libfreetype6-dev libpng-dev \
+     && apt-get -y install --no-install-recommends libcurl4-openssl-dev rapidjson-dev libfreetype6-dev libpng-dev libeigen3-dev \
```

Comment:
One thing I did not explicitly mention in the previous comment is that depending on the OS release libeigen3 may differ in what version of Eigen it is (I believe that is very rare or no longer the case nowadays but it used to be true for ubuntu releases). 

For [Debian bookworm it looks like this is Eigen 3.4.0-4](https://packages.debian.org/bookworm/armel/libeigen3-dev).

Are there any issues with this? (I assume not since we rely on an older version iirc but I just wanted to mention it to be extra sure)

Reply:
Functionality-wise, the version of Eigen should not matter at all, we only use very, very little functionality of Eigen which is absolutely uncritical. What has proved to be critical in the past - there have been versions of Eigen3 which did not compile (on some compilers), so I'd think - if it compiles (and unit-tests work), then it's fine here.